training/nvidia-bootc/Containerfile

```diff
@@ -99,6 +99,7 @@ COPY --from=builder /home/builder/yum-packaging-precompiled-kmod/RPMS/*/*.rpm /r
 COPY --from=builder --chmod=444 /home/builder/yum-packaging-precompiled-kmod/tmp/firmware/*.bin /lib/firmware/nvidia/${DRIVER_VERSION}/
 # Temporary workaround until the permanent fix for libdnf is merged
 COPY nvidia-toolkit-firstboot.service /usr/lib/systemd/system/nvidia-toolkit-firstboot.service
+COPY add_ilab_wrapper.sh add_ilab_wrapper.sh
```

Comment:
Could you do the script below as inside of the run command?
```
if [ -f /run/.input/ilab" ]; then \
	cp /run/.input/ilab /usr/local/bin/ilab \
else \
	curl -o /usr/local/bin/ilab "https://raw.githubusercontent.com/containers/ai-lab-recipes-upstream/main/training/ilab-wrapper/ilab" \
fi
```

Reply:
Sure. Much better. I'm updating it.